src/instructlab/utils.py

```diff
@@ -514,19 +514,89 @@ def read_taxonomy(_logger, taxonomy, taxonomy_base, yaml_rules):
     return seed_instruction_data
 
 
-def get_ssl_cert_config(tls_client_cert, tls_client_key, tls_client_passwd):
-    if tls_client_cert:
-        return tls_client_cert, tls_client_key, tls_client_passwd
+def endpoint_tls_options(f):
+    """Add TLS options for endpoints"""
+    options = [
+        # TLS parameters are not exposed as function arguments. Instead they
+        # are directly read from `ctx.params`
+        click.option(
+            "--endpoint-ca-cert",
+            "ca_certfile",
+            type=click.Path(),
+            help="Path to a CA cert bundle file for OpenAI endpoint.",
```

Comment:
```suggestion
            help="Path to a PEM-encoded CA cert bundle file for OpenAI endpoint.",
```

Reply:
Should we make it clear that this is the expected format?

Reply:
Python can load DER encoded certificates, too.

Reply:
It doesn't add much useful information. Non experts will rarely encounter a different format. Experts can deduct from the parameter names that it cannot be a PKCS11 URI or PKCS12 file. PKCS12 would not ask for a key file and instead have an option to specify a cert name in the archive.

Reply:
Ok, fair enough!